tests/test_utils/tests.py

```diff
@@ -1113,6 +1114,21 @@ def test_not_equal_parsing_errors(self):
         with self.assertRaises(AssertionError):
             self.assertJSONNotEqual(valid_json, invalid_json)
 
+    def test_method_frames_ignored_by_unittest(self):
+        try:
+            self.assertJSONEqual("1", "2")
+        except AssertionError:
+            exc_type, exc, tb = sys.exc_info()
+
+        result = unittest.TestResult()
+        result.addFailure(self, (exc_type, exc, tb))
+        stack_summary = traceback.extract_tb(exc.__traceback__)
```

Comment:
If you agree to rename `stack` in the messages app, I would suggest we also rename here.